crates/wasm-mutate/src/mutators/rename_export.rs

```diff
@@ -27,7 +31,12 @@ impl RenameExportMutator {
                 };
                 String::from(s)
             }
+        };
+        // Add one symbol at a time until it is not contained in the export field names
+        while info.export_names.contains(&s.as_str()) {
+            s.push_str(&self.limited_string(rnd, info, 2));
```

Comment:
Two things:

1. We should use a `HashSet<String>` for `export_names` so that we can do `O(1)` lookups instead of `O(n)` lookups.

2. Because `limited_string` returns a `String`, we are doing a heap allocation that we then immediately throw away after pushing onto `s`. And we do that for every iteration of the loop. It would be better if we made a specialized version of `limited_string` that reuses an existing allocation by taking a `s: &mut String` parameter that it mutates directly instead of returning a new `String` allocation.

Reply:
Done